app/models/candidate_interface/contact_details_form.rb

```diff
@@ -5,6 +5,15 @@ class ContactDetailsForm
     attr_accessor :phone_number, :address_line1, :address_line2, :address_line3,
                   :address_line4, :postcode
 
+    validates :address_line1, :address_line3, :postcode, presence: true, on: :address
+
+    validates :address_line1, :address_line2, :address_line3, :address_line4,
+              length: { maximum: 200 }, on: :address
```

Comment:
Can you double check the API docs? I believe they may have a smol-er limit, like 100 iirc

Reply:
Currently the each address field has maxLength of 50. 
I'm fine with increasing this if needs be 👍

Reply:
I'll update it to match the API spec.